specification/tasks/get/GetTaskRequest.ts

```diff
@@ -26,7 +26,7 @@ import { Duration } from '@_types/Time'
  * Returns information about the tasks currently executing in the cluster.
  * @rest_spec_name tasks.get
  * @availability stack since=5.0.0 stability=experimental
- * @availability serverless stability=experimental visibility=public
+ * @availability serverless stability=experimental visibility=private
```

Comment:
```suggestion
 * @availability serverless stability=experimental visibility=public
```

`tasks.get` *is* public, but `tasks.list` is not, but this is already correct: https://github.com/elastic/elasticsearch-specification/blob/afcfe48dedd1e40515261117540927d3a90df15b/specification/tasks/list/ListTasksRequest.ts#L29

Reply:
I finally found why this wasn't making sense to me :sweat_smile: 
The Ruby code generator had a bug and `task.get` didn't have `id` as required so I was testing the `tasks.list` path (without an id) :facepalm: